src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -156,27 +156,29 @@
 			</ol>
 		</section>
 
-		<section class="statistics section text-center">
-			<div class="row">
-				<div class="small-12 columns">
-					<kv-responsive-image
-						class="statistics__img"
-						:images="statistics"
-						loading="lazy"
-						alt=""
-					/>
-				</div>
-				<div class="small-12 columns">
-					<h2 class="statistics__header">
-						<span>100%</span>
-						<span class="statistics__header-small">of your loan goes to the field.</span>
-					</h2>
-					<p class="statistics__body">
-						We don't take a penny.
-					</p>
-				</div>
-				<homepage-statistics />
+		<section class="statistics section">
```

Comment:
Is this block meant to show always? Seems like it may be duplicated content if `statisticsContentfulContentGroup` exists.

Reply:
There is some additional work here, but replacing this 100% loan image and moving it into contentful is handled in this ticket: [Grow-432](https://kiva.atlassian.net/browse/GROW-432)

Once this is brought over to contentful, this top section will be removed and data will be coming through statisticsContentfulContentGroup.